### PR TITLE
[CRIMAPP-1259] Allow cifc submission for appeal summary case

### PR DESCRIPTION
**Pull request by mseedat-moj**
## Description of change
Allows CIFC application to be submitted when appeal summary case

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-1259

## Notes for reviewer
- Requires more specs 
- Hides codefendant and appeal reference output

### CODE REVIEW

File: app/validators/appeal_details/answers_validator.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ def validate
     end
 
     def complete?
+      return true if record.cifc?
```
Comment (timpeat) [inline app/validators/appeal_details/answers_validator.rb:21]: Would it be better to add this guard to the '#applicable?' method?

File: app/presenters/summary/sections/codefendants.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ module Summary
   module Sections
     class Codefendants < Sections::BaseSection
       def show?
-        kase.present? && super
+        kase.present? && (crime_application.application_type != 'change_in_financial_circumstances') && super
```
Comment (timpeat) [inline app/presenters/summary/sections/codefendants.rb:5]: Any reason not use use .cifc? here?

File: app/validators/appeal_details/answers_validator.rb (hunk 1 above)
Comment (hiboabd) [reply app/validators/appeal_details/answers_validator.rb:21]: @mseedat-moj Is this correct? as the `appeal_original_app_submitted` question could have been answered in a cifc application?

File: app/presenters/summary/sections/case_details.rb [hunk 3]
```diff
@@ -36,13 +36,13 @@ def answers
 
           Components::ValueAnswer.new(
             :appeal_financial_circumstances_changed, kase.appeal_financial_circumstances_changed,
-            show: original_app_submitted?,
+            show: original_app_submitted? && !crime_application.cifc?,
```
Comment (hiboabd) [inline app/presenters/summary/sections/case_details.rb:39]: Would it be cleaner to put a guard in the `original_app_submitted` and `financial_circumstances_changed` questions?

**Review by hiboabd — APPROVED** [review]

**Pull request merged**